src/index.html

```diff
@@ -359,9 +359,18 @@ <h3>Some example of queries include:</h3>
                             <p>* To fetch all the variants in a segment region:
                                 <br/> <a href="http://www.ebi.ac.uk/eva/webservices/rest/v1/segments/11:128446-128446/variants?species=hsapiens_grch37" target="_blank">http://www.ebi.ac.uk/eva/webservices/rest/v1/segments/11:128446-128446/variants?species=hsapiens_grch37</a></p>
 
-                            <p><br/> * To fetch all the info of a variant:
+                            <p><br/> * To fetch all the info for a variant by ID:

```

Comment:
The PR is fine right now, although I suggest changing the first line that says 
```
The general structure of a EVA REST web service URL is:
http://www.ebi.ac.uk/eva/webservices/rest/{version}/{category}/IDs/{resource}?{filters}
```
to
```
The general structure of a EVA REST web service URL is one of:
http://www.ebi.ac.uk/eva/webservices/rest/{version}/{category}/IDs/{resource}?{filters}
http://www.ebi.ac.uk/eva/webservices/identifiers/{version}/{category}/ID
```
or something similar